app/templates/components/change-password-form.hbs

```diff
@@ -1,37 +1,30 @@
-<div class="form-group">
-  <label for="previous">Current Password</label>
-  {{input type="password"
-          class="form-control"
-          id="previous"
-          placeholder="Old password"
-          value=passwords.previous}}
 
-</div>
+<h4>Change your password! </h4>
 
-<div class="form-group">
-  <label for="next">New Password</label>
-  {{input type="password"
-          class="form-control"
-          id="next"
-          placeholder="New password"
-          value=passwords.next}}
-</div>
+  {{#if isEngineer}}
+      {{#link-to 'engineer'}} {{#paper-button fab=true}}Back{{/paper-button}}{{/link-to}}
```

Comment:
On line 8 you have a very similar block of code that is structured in a different way. Lets make these uniform so that its easier to spot duplication. I am a fan of the style below that uses multi-lines, its easier to digest when reading.